.github/workflows/main.yaml

```
@@ -0,0 +1,20 @@
+name: "Continuous Build"
+
+on:
```

You might also want to add `workflow_dispatch` to be able to run the build manually:
```suggestion
on:
  workflow_dispatch:
```